src/pages/workspaces/List.js

```diff
@@ -262,7 +262,7 @@ export const WorkspaceList = () => {
     _.filter(ws => {
       const { workspace: { namespace, name } } = ws
       return Utils.textMatch(filter, `${namespace}/${name}`) &&
-        (_.isEmpty(accessLevelsFilter) || accessLevelsFilter.includes(ws.accessLevel)) &&
+        (_.isEmpty(accessLevelsFilter) || _.some(accessLevel => Utils.hasAccessLevel(accessLevel, ws.accessLevel), accessLevelsFilter)) && //accessLevelsFilter.includes(ws.accessLevel)) &&
```

Comment:
If we're going to do this, we should change the behavior of the `Select`, since it would no longer make sense to choose multiple of these or select 'no access'. Also, because these values are exposed in the URL, we should add some defensive logic to make sure we handle/ignore old values coming in from people's bookmarks.

Reply:
Argh. This feels familiar now. I think we've had this exact conversation before. I don't disagree with your points, but I also don't agree with the current behavior. Maybe this needs some UX attention.